Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/FileIntegrity.cs

```diff
@@ -2,14 +2,17 @@
 {
     public class FileIntegrity
     {
-        public FileIntegrity(string fileName, FileIntegrityStatus status)
+        public FileIntegrity(string fileName, FileIntegrityStatus status, string message = null)
         {
             FileName = fileName;
             Status = status;
+            Message = message;
         }
 
         public string FileName { get; private set; }
 
+        public readonly string Message;
```

Comment:
I don't see any read usage of `Message` field within this PR. Is this a mistake?

Reply:
Yup, and it's corrected now